exporter/elasticsearchexporter/model_test.go

```diff
@@ -218,6 +218,61 @@ func TestEncodeEvents(t *testing.T) {
 	}
 }
 
+func TestEncodeLogECSModeDuplication(t *testing.T) {
+	resource := pcommon.NewResource()
+	err := resource.Attributes().FromRaw(map[string]any{
+		semconv.AttributeServiceName:    "foo.bar",
+		semconv.AttributeHostName:       "localhost",
+		semconv.AttributeServiceVersion: "1.1.0",
+		semconv.AttributeOSType:         "darwin",
+		semconv.AttributeOSDescription:  "Mac OS Mojave",
+		semconv.AttributeOSName:         "Mac OS X",
+		semconv.AttributeOSVersion:      "10.14.1",
+	})
+	require.NoError(t, err)
+
+	expectedDocFields := pcommon.NewMap()
+	err = expectedDocFields.FromRaw(map[string]any{
+		"service.name":     "foo.bar",
+		"service.version":  "1.1.0",
+		"agent.name":       "otlp",
+		"host.os.platform": "darwin",
+		"host.os.full":     "Mac OS Mojave",
+		"host.os.name":     "Mac OS X",
+		"host.os.version":  "10.14.1",
+		"host.os.type":     "macos",
+		"host.name":        "localhost",
+	})
```

Comment:
Maybe I'm missing something obvious, but is `expectedDocFields` being used in this test?

Reply:
yes they are not. leftovers